pkgs/by-name/ba/bankstown-lv2/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, lv2
+, pkg-config
+, rustPlatform
+, fetchFromGitHub
+, fetchpatch
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bankstown-lv2";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "chadmed";
+    repo = "bankstown";
+    rev = version;
+    hash = "sha256-IThXEY+mvT2MCw0PSWU/182xbUafd6dtm6hNjieLlKg=";
+  };
+
+  cargoSha256 = "sha256-yRzM4tcYc6mweTpLnnlCeKgP00L2wRgHamtUzK9Kstc=";
+
+  installPhase = ''
+    export LIBDIR=$out/lib
+    mkdir -p $LIBDIR
+
+    make
+    make install
+  '';
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    lv2
+  ];
```

Comment:
As far as I can tell, bankstown doesn't actually require `lv2` or `pkg-config` to find it. It uses the `lv2` create, which vendors the headers it uses (see https://github.com/RustAudio/rust-lv2/tree/develop/sys) and all other code is pure Rust.

Reply:
That it doesn't require lv2 doesn't mean it should not have lv2 as an input.
But in this case it indeed does not link against lv2, and the output doesn't reference lv2.
So we can drop the dependency.
Thanks

Reply:
True, that was phrased a bit poorly by me. I meant that it does not link against it and merely uses vendored headers to provide the needed C-API to be loadable as an lv2 plugin. Thanks for change and for having a look again yourself :)